### DISCUSSION TITLE
How can I tell if the Draco decoder is being loaded?

### DISCUSSION THREAD

**Discussion by rebrob** [google/model-viewer#4937] [Q&A]
Hello,

I saw somewhere that there should be a console message if the Draco decoder has been loaded and running. I have tested with a .glb file with confirmation that the draco extension is being used. ("extensions":{"KHR_draco_mesh_compression")

I am not, however, seeing any indication in the console that the Draco decoder is being implemented in model-viewer.

Does model-viewer support Draco compression in both glb and gITF files?

Could you show a screen shot of where in the console I should see indication of the Draco decoder being successfully loaded?

Thanks,
Rebecca

**Comment by elalish** [comment]
I don't know about a console message, but you know if the Draco decoder is loaded because you can see the object. If it's draco encoded, it can't be opened unless you have a decoder. You can set where the decoder is downloaded from here if you don't want to use our default CDN: https://modelviewer.dev/docs/#entrydocs-loading-staticproperties-dracoDecoderLocation

**Reply by rebrob** [reply]
Thanks! That's very helpful.
